connector/connect/common/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -23,7 +23,6 @@ import "google/protobuf/any.proto";
 import "spark/connect/expressions.proto";
 import "spark/connect/types.proto";
 import "spark/connect/catalog.proto";
-import "spark/connect/common.proto";
```

Comment:
Do you know when it becomes unused?

Reply:
Sorry for the late reply. This import has been an unused import since it was added to `spark/connect/relations.proto`:

https://github.com/apache/spark/pull/45232/files#diff-5b26ee7d224ae355b252d713e570cb03eaecbf7f8adcdb6287dc40c370b71462R26